src/08-js-for-of-loop.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * This quiz is to test your knowledge of for-of-Loop.
+ *
+ * 1. When you finish the quiz, create a PR on you repository
+ * 2. Submit to your lead
+ */
+
+// Question 1: Loop through the array and print the names of the Apostles
+const apostles = ["Peter", "James", "John", "Andrew", "Philip"];
+
+for (const apostle of apostles) {
+  console.log("[Question #1]: ", apostle);
+}
+
+// Answer should be: Peter, James, John, Andrew, Philip (each in a new line)
+
+// Question 2: Calculate the sum of the ages of the Patriarchs
+const patriarchAges = [930, 912, 905, 895];
+let sumAges = 0;
+
+for (const smAge of patriarchAges) {
+  sumAges += smAge;
+}
+console.log("[Question #2]: ", sumAges);
+// console.log(answer) // Answer should be: 3642
+
+// Question 3: Print the plagues of Egypt
+const plagues = ["Frogs", "Locusts", "Hail"];
+
+for (const plague of plagues) {
+  console.log("[Question #3]: ", plague);
+}
+// Answer should be: Frogs, Locusts, Hail (each in a new line)
+
+// Question 4: Convert Bible books to uppercase
+const books = ["Genesis", "Exodus", "Leviticus"];
+
+for (const book of books) {
+  const uppercaseBooks = book.toUpperCase();
+  console.log("[Question #4]: ", uppercaseBooks);
+}
+
+// Answer should be: GENESIS, EXODUS, LEVITICUS (each in a new line)
+
+// Question 5: Count the number of times 'apple' appears in the array
+const fruitsInEden = ["apple", "pear", "apple", "grape"];
+let count = 0;
+
+for (let i = 0; i < fruitsInEden.length; i++) {
```

Comment:
solo se permite usar 
`for-of` ningun for loop normal. Favor cambiar todos los lugare donde se use el for normal por un `for-of` que es de lo que trata este quiz